solutions/observability/apm/apm-server/advanced-setup.md

```diff
@@ -11,7 +11,11 @@ products:
 
 # APM Server advanced setup [apm-setting-up-and-running]
 
-Before reading this section, refer to [](/solutions/observability/apm/apm-server/setup.md) for basic installation and running instructions.
+Before reading this section, refer to [](/solutions/observability/apm/apm-server/setup.md) for basic installation and deployment instructions.
+
+:::{note}
+This section covers {{apm-server}} setup. We recommend using the [{{edot}} Collector](opentelemetry://reference/index.md) as the preferred way to collect application telemetry data using OpenTelemetry. For mobile applications, we recommend using the [EDOT Android SDK](apm-agent-android://reference/edot-android/index.md) or [EDOT iOS SDK](apm-agent-ios://reference/edot-ios/index.md).
+:::
```

Comment:
The mobile applications note here feels a little out of place as it seems unrelated to the APM Server. But I don't have a strong opinion.